client/rtcClient.js

```diff
@@ -63,21 +63,15 @@ function loginSuccess () {
       participants: $scope.roomUsers,
       meetings: $scope.roomName,
       meeting: $scope.roomName,
-      meetingUrl: location.href
-    })
-  }).catch(function (err) {
-    console.log('ERROR:', err)
-  }).then(function (result) {
-    console.log('meeting result:', result)
-    return app.service('participants').patch(easyrtc.myEasyrtcid, {
+      meetingUrl: location.href,
```

Comment:
sorry, not being specific -- why is there a meetingUrl being patched to a participants object? IIRC meetingUrl is only something that exists in a meeting object, shouldn't this break things?